test/unit-tests/org/orgTest.js

```diff
@@ -53,7 +59,16 @@ describe('Test Org Controller', () => {
                         done(err)
                       }
 
-                      done()
+                      Org.findOneAndUpdate()
+                      .byShortName(orgWithZeroIdQuota.shortName)
```

Comment:
Is odd that tests pass when using `shortName` since this field does not exist in the Org model. This has been fixed in PR #52. 

Change all occurrences of `shortName` in favor of `short_name` in "orgTest.js" file